### PR TITLE
sending from a new branch named new

### LINKED ISSUE #4: Aditya - HTML Basics Remake
@adithyaRocks Tag this when sending in your pull request.

### PR DESCRIPTION
**Pull request by AdithyaSuresh1501**
Fixes #4 

## Proposed Changes- changed all

<a href="https://gitpod.io/#https://github.com/aravindvnair99/ACM-Amrita-2019-2020-Assignments/pull/50"></a>

### CODE REVIEW

File: Adithya/HTMLbasicsremake/form.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        <label for="Last Name">Last Name</label>
+        <input type="text"  name="Last Name" placeholder="Last Name" required autocomplete="name">
+        </p>
+        <p>
+        Gender:Male:
+        <input type="radio" name="gender" value="Male" autocomplete="sex">
+        Female:
+        <input type="radio" name="gender" value="Female" autocomplete="sex">
+        Other:
+        <input type="radio" name="gender" value="Other" autocomplete="sex">
```
Comment (aravindvnair99) [inline Adithya/HTMLbasicsremake/form.html]: - Missing `label`.
- Radio inputs are not compatible with autocomplete.

File: Adithya/HTMLbasicsremake/form.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    <title>Form</title>
+</head>
+<body>
+    <h1>Register</h1>
+    <form>
+        <p>
+        <label for="First Name">First Name</label>
+        <input type="text"  name="First Name" placeholder="First Name" required autocomplete="name">
+        <label for="Last Name">Last Name</label>
+        <input type="text"  name="Last Name" placeholder="Last Name" required autocomplete="name">
```
Comment (aravindvnair99) [inline Adithya/HTMLbasicsremake/form.html]: Wrong `autocomplete`. Refer this: [The HTML autocomplete attribute
](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete)

File: Adithya/HTMLbasicsremake/form.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        Other:
+        <input type="radio" name="gender" value="Other" autocomplete="sex">
+        </p>
+        <p>
+        <label for="Email">Email:</label>
+        <input type="email" name="Email" placeholder="Email" required autocomplete="email" >
+        <label for="Password">Password:</label>
+        <input type="password" name="Password" minlength="5" maxlength="15" required autocomplete="new-password">
+        <label for="birthday">Birthday</label>
+        <input type="date" name="Birthday"><br>
```
Comment (aravindvnair99) [inline Adithya/HTMLbasicsremake/form.html]: Missing `autocomplete`.

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]

File: Adithya/HTMLbasicsremake/form.html (hunk 1 above)
Comment (AdithyaSuresh1501) [reply Adithya/HTMLbasicsremake/form.html]: label wasnt required here right

File: Adithya/HTMLbasicsremake/Pokemon .html [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    <th>Image</th>
+    <th>name</th>
+    <th>Type</th>
+    <th>Evolves Into</th>
+</tr>
+<tr>
+    <th><img src="001Bulbasaur.png"> </th>
+    <th>Bulbassaur</th>
+    <th>Grass/Poison</th>
+    <th><a href="http://pokemon.wikia.com/wiki/Ivysaur">Ivyasaur</a></th>
```
Comment (VinayakTekade) [inline Adithya/HTMLbasicsremake/Pokemon .html]: Use `th` is used only in the first row. Use `td` for other rows.
`th` stands for table heading and `td` stands for table definition.

File: Adithya/HTMLbasicsremake/Pokemon .html [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+    
+</tr>
+    
+<tr>
+    <th><img src="004Charmander.png"></th>
+    <th>Charmander</th>
+    <th>Fire</th>
+    <th><a href="http://pokemon.wikia.com/wiki/Charmeleon">Charmeleon</a></th>
```
Comment (VinayakTekade) [inline Adithya/HTMLbasicsremake/Pokemon .html]: Use `td`

File: Adithya/HTMLbasicsremake/Pokemon .html [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+    
+</tr>
+    
+<tr>
+    <th><img src="007Squirtle.png"></th>
+    <th>Squirtle</th>
+    <th>Water</th>
+    <th><a href="http://pokemon.wikia.com/wiki/Wartortle">Wartortle</a></th>
```
Comment (VinayakTekade) [inline Adithya/HTMLbasicsremake/Pokemon .html]: Use `td`

File: Adithya/HTMLbasicsremake/form.html [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Form</title>
+</head>
+<body>
+    <h1>Register</h1>
+    <form>
+        <p>
+        <label for="First Name">First Name</label>
+        <input type="text"  name="First Name" placeholder="First Name" required autocomplete="given-name">
```
Comment (VinayakTekade) [inline Adithya/HTMLbasicsremake/form.html]: `for` points to an `id`. See the solution in my folder and use the same values in `for` and `id`

File: Adithya/HTMLbasicsremake/form.html [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+        <label for="Last Name">Last Name</label>
+        <input type="text"  name="Last Name" placeholder="Last Name" required autocomplete="family-name">
```
Comment (VinayakTekade) [inline Adithya/HTMLbasicsremake/form.html]: Use `id`

File: Adithya/HTMLbasicsremake/form.html [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+        </p>
+        <p>
+        Gender: 
+        <labe for="male">Male:</labe>
+        <input type="radio" name="gender" value="Male" >
```
Comment (VinayakTekade) [inline Adithya/HTMLbasicsremake/form.html]: Use `id`

File: Adithya/HTMLbasicsremake/form.html [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+        <labe for="female">Female:</labe>
+        <input type="radio" name="gender" value="Female" >
```
Comment (VinayakTekade) [inline Adithya/HTMLbasicsremake/form.html]: Use `id`

File: Adithya/HTMLbasicsremake/form.html [hunk 11]
```diff
@@ -0,0 +1,41 @@
...
+        <labe for="other">Other:</labe>
+        <input type="radio" name="gender" value="Other">
```
Comment (VinayakTekade) [inline Adithya/HTMLbasicsremake/form.html]: Use `id`

File: Adithya/HTMLbasicsremake/form.html [hunk 12]
```diff
@@ -0,0 +1,41 @@
...
+        </p>
+        <p>
+        <label for="Email">Email:</label>
+        <input type="email" name="Email" placeholder="Email" required autocomplete="email" >
```
Comment (VinayakTekade) [inline Adithya/HTMLbasicsremake/form.html]: Use `id`

File: Adithya/HTMLbasicsremake/form.html [hunk 13]
```diff
@@ -0,0 +1,41 @@
...
+        <label for="Password">Password:</label>
+        <input type="password" name="Password" minlength="5" maxlength="15" required autocomplete="new-password">
```
Comment (VinayakTekade) [inline Adithya/HTMLbasicsremake/form.html]: Use `id`

File: Adithya/HTMLbasicsremake/form.html [hunk 14]
```diff
@@ -0,0 +1,41 @@
...
+        <label for="birthday">Birthday</label>
+        <input type="date" name="Birthday" autocomplete="bday"><br>
```
Comment (VinayakTekade) [inline Adithya/HTMLbasicsremake/form.html]: Use `id`

File: Adithya/HTMLbasicsremake/form.html [hunk 15]
```diff
@@ -0,0 +1,41 @@
...
+        <label for="Check">I agree to terms and conditions</label>
+        <input type="checkbox" name="Check" value="check" required><br>
```
Comment (VinayakTekade) [inline Adithya/HTMLbasicsremake/form.html]: Use `id`

**Review by VinayakTekade — DISMISSED** [review]

**Comment by VinayakTekade** [comment]
Make the changes and send another pull request.

**Review by aravindvnair99 — COMMENTED** [review]
@adithyaRocks Merging and closing the pull request for now. Make the necessary changes in another pull request and send it over from a different branch. Also, make sure to fill up the pull request template and also refactor the code henceforth.

**Pull request merged**